libs/ngx-dsfr/footer/src/lib/footer.component.ts

```diff
@@ -68,6 +61,10 @@ export class DsfrFooterComponent implements OnInit, OnChanges {
     if (this.operatorLogoSrc && !this.operatorLogoAlt) {
       throw EMPTY_ALT_ERROR;
     }
+
+    if (!this.copyright) {
+      throw EMPTY_COPYRIGHT_ERROR;
```

Comment:
You don't need that. As I said in my previous comment, just give the `copyright` property a default value of the one suggested in the doc.

I don't think it will change a lot or that it's the kind of thing developers will really think about.

Don't forget to delete the error.